spec/index.html

```diff
@@ -748,7 +773,32 @@ <h2>Literals</h2>
     <p>denote the same <a data-lt="literal value">value</a>, but are not the
       same literal <a>RDF terms</a> and are not
       <a>term-equal</a> because their
-      <a>lexical form</a> differs.</p>
+      <a>lexical forms</a> differ.</p>
+
+    <section id="section-text-direction" class="informative">
+      <h3>Initial Text Direction</h3>
+
+      <p>The <a>base direction</a> of a <a>directional language-tagged string</a>
+        provides a means of establishing the initial direction of text,
+        including text which is mixture of right-to-left and left-to-right scripts.
+        The [[[?UAX9]]] [[?UAX9]] provides support for automatically rendering
+        a sequence of characters in logical order,
+        so that they are visually ordered as expected.
+        However, this is not sufficient to correctly render bidirectional text.</p>
+
+      <p>For example, the text `"פעילות הבינאום, W3C"` with language tag "he" (for Hebrew) may be displayed
+        incorrectly unless the <a>base direction</a> of `rtl` (for right-to-left)
+        is also provided.
+        When this is provided to a user agent,
+        for example using the HTML `dir` attribute,
+        it would be correctly presented as follows: <div lang="he" dir="rtl">פעילות הבינאום, W3C</div></p>
+
+      <p class="note">The absence of a <a>base direction</a> does not necessarily imply that
+        the text has no initial text direction;
+        as described in [[[?UAX9]]],
+        strings may be embedded within structures which establish an <em>embedding direction</em>,
+        which determines the default bidirectional orientation of text.</p>
```

Comment:
Within the context of our specifications, this note really isn't very useful. The only syntax we define where it could possibly apply is RDF/XML, for which XML 1.1 doesn't support base direction.

```suggestion
```